ice/src/agent/agent_selector.rs

```diff
@@ -61,33 +61,33 @@ trait ControlledSelector {
 }
 
 impl AgentInternal {
-    async fn is_nominatable(&self, c: &Arc<dyn Candidate + Send + Sync>) -> bool {
-        let start_time = self.start_time.lock().await;
+    fn is_nominatable(&self, c: &Arc<dyn Candidate + Send + Sync>) -> bool {
+        let start_time = { *self.start_time.lock().unwrap() };
```

Comment:
could you add a comment stating why we'll never panic on unwrap here? also in general `expect("..")` is better in production code (unwrap is ok for tests)

Reply:
Done [87fca0e](https://github.com/webrtc-rs/webrtc/pull/338/commits/87fca0e4031fde7b8a9cecf37e1878c2dc2096f1)